company/okrs/2020_q1.md

```diff
@@ -0,0 +1,106 @@
+   1. Get 70% of companies on a free trial to convert to paid customers.
+1. **Sales (Pipeline Optimization): Scale our sales pipeline to 50+ deals per AE, of which 60% of deals are engaged and actively evaluating Sourcegraph.**
+   1. Round robin new instances across team to collectively achieve a 15%+ response rate from Prospects who set up a new Sourcegraph instance.
+
+1. **VP Business: Build a Customer Success organization that supports and helps us exceed our ARR goals.**
+   1. Generate at least [$M][M] in net new ARR from existing accounts.
+   1. Hire a Head of Customer Success who can own this function.
+   1. Reach monthly NPS of 60 at all Tier 1 customers.
+
+1. **VP Business: Make Sourcegraph a lean organization that uses budgets to guide decision making.**
+
+1. **VP Marketing: Deliver sufficient MQLs to sales to reach [$N][N] net new ARR goal.**
+1. **VP Marketing: Create new category of developer tool.**
+1. **VP Marketing: Become the dominant provider of source code search.**
+
+## Product
+
+1. **Product: Create a customer driven product process**
+   1. All product engineering work has a delivery plan (where "delivery plan" links to a new handbook page that defines it, based on [RFC 97](https://docs.google.com/document/d/1cZ7JIVuRWrF2MxwDdH36SC7zOyT2qJf9AMUd9Wc9_aY/edit#heading=h.dkow31it1qt1)).
+   1. Handbook describes the lifecycle of a feature and who is responsible for each part of the lifecycle.
+   1. Handbook describes product planning processes and API between product and engineering.
+1. **Product: Build the product and design team to support engineering and ARR growth**
+   1. Hire +1 UX Designer.
+   1. Hire +1 Product Manager.
+
+## Engineering
+
+### VP Eng
+
+1. **Build a world class engineering team.**
+   1. Grow team to plan (+1 code intel, +2 distribution, +2 web, +1 core services).
+   1. Transition an existing teammate to be (or hire) an Engineering Manager of the distribution team.
+   1. Document how we do performance reviews and determine compensation for teammates in engineering roles.
+1. **Create a data driven planning process.**
+   1. Each team commits to a written work plan for each release cycle.
+   1. Each team completes >80% of the work that they committed to at the beginning of the release cycle each iteration.
+
+### Distribution
+
+1. **Resolve issues for major customers**
+   1. Ensure that no technical issue blocks a deal for any [tier 1 or tier 2 customer](https://sourcegraph.looker.com/dashboards/132).
+1. **Improve developer productivity**
+   1. Automate away release captain role and adopt continuous releases (no human in loop required by default).
+1. **Improve installation and site admin onboarding process**
+   1. Track time to important admin setup milestones (i.e. instance started, admin account created, >1 repositories added) are tracked in [pings](https://docs.sourcegraph.com/admin/pings).
```

Comment:
@beyang Dan said that we already track (or can track this based on current pings): https://github.com/sourcegraph/about/pull/422#discussion_r371519877

If so (can you confirm?) I think we can delete this KR.

```suggestion
```

Reply:
We track _some_ things. Here's an example of major milestones we already have data on https://sourcegraph.looker.com/looks/508. @beyang whether or not you leave it as a KR, it'd probably be worth it to chat and make sure we can get you the data you need.